addons/website/static/src/interactions/zoomed_background_shape.edit.js

```diff
@@ -0,0 +1,6 @@
+import { registry } from "@web/core/registry";
+import { ZoomedBackgroundShape } from "./zoomed_background_shape";
+
+registry
+    .category("website.edit_active_elements")
+    .add("website.zoomed_background_shape", ZoomedBackgroundShape);
```

Comment:
Since it's identical, maybe just do this in the other file? :shrug:

Reply:
I am in favor of not including it in the visitor's code.